.github/workflows/renovate-validate.yaml

```diff
@@ -0,0 +1,12 @@
+name: Validate renovate config
+on:
+  pull_request: {}
+jobs:
+  validate:
+    runs-on: ubuntu-latest
+    timeout-minutes: 5
+    steps:
+      - name: Checkout Code
+        uses: actions/checkout@8ade135a41bc03ea155e62e844d188df1ea18608 # v4.1.0
+      - run: |-
+          docker run -i --rm -v $(realpath .):/repo -w /repo --entrypoint renovate-config-validator ghcr.io/renovatebot/renovate --strict
```

Comment:
oh, run, we are validating an old renovate configuration file with a new renovate version.

Doesn't this work with the github action?

Reply:
It doesn't seem possible sadly :(

The `docker-cmd-file` is automatically mounted and `basename`d:
https://github.com/renovatebot/github-action/blob/044641a5b884facaf733f679cc57474321fa039d/src/renovate.ts#L42-L49

I can see your concern of these getting out of sync though. I can add a regex to #768 so renovate updates both things (action and this image) it that looks good to you.

Reply:
sure!

Reply:
@mem Any other thoughts? I can make the change we were discussing here in #768 if that's fine by you 😊